content/cli/orch_cli/blueprints.md

```diff
@@ -456,3 +479,172 @@ Blueprint `cloudify-nodecellar-example` was set to global
 
 ...
 {{< /highlight >}}
+
+
+### filters
+
+A filter is defined as a set of filter-rules that can be used to filter a list of blueprints, based on their labels and certain attributes.
+Blueprints can be filtered by the attribute `created_by`.
+For more information regarding the meaning of each filter rule, please refer to the [filter-rules document]{{< relref "cli/orch_cli/filter-rules.md" >}}.
+
+#### filters create
+
+##### Usage
+
+`cfy blueprints filters create [OPTIONS] FILTER_ID` 
+
+Create a new blueprints' filter.
+
+`FILTER-ID` is the new filter's ID
+
+##### Optional flags
+
+* `--labels-filter TEXT`    A list of labels' filter rules separated
+                            with an `and`. Labels' filter rules must be
+                            one of: `<key>=<value>`, `<key>!=<value>`, `<key>
+                            is null`, `<key> is not null`. `<value>` can be a
+                            single string, or a list of strings of the
+                            form `[<value1>,<value2>,...]`. E.g. `"a=b and
+                            c!=[d,e] and f is not null"`. The labels'
+                            filter rules will be saved in lower case.
+  
+* `--attrs-filter TEXT`     A list of attributes' filter rules separated
+                            with an `and`. Attributes' filter rules must
+                            be one of:  `<key>=<value>`, `<key>!=<value>`,
+                            `<key> contains <value>`, `<key> does-not-
+                            contain <value>`, `<key> starts-with <value>`,
+                            `<key> ends-with <value>`. `<key> is not empty`.
+                            `<value>` can be a single string, or a list of
+                            strings of the form `[<value1>,<value2>,...]`.
+                            Allowed attributes to filter by are:
+                            [blueprint_id, created_by, site_name, schedules]. 
```

Comment:
Looks like these are the attributes for deployment filters, rather than blueprints'

Reply:
Right, changed.